src/web/mjs/engine/Connector.mjs

```diff
@@ -743,4 +743,11 @@ export default class Connector {
             return;
         }
     }
+
+    _getFormatRegex() {
```

Comment:
Consider making this a getter, so it can be accessed like a property
```javascript
get formatRegex() {
    return // ...
}
```

Reply:
Never mind, i see that this should probably be overwritten by specialisation of this class...